Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -214,6 +221,11 @@ private void Fire(InputAction.CallbackContext ctx)
         StartCoroutine(UnpressTrigger());
     }
 
+    private void UpdateHudLeap(Rigidbody body)
```

Comment:
Iffy name, just UpdateSpeedLines instead?

(also, it feels like the HUD should be subscribing to these events itself)

Reply:
Updated name, I'll leave the scope thing for refactoring issue